Source/Scene/Scene.js

```diff
@@ -822,6 +822,19 @@ define([
         },
 
         /**
+         * Gets an event that's raised when the terrain provider is changed
+         * @memberof Scene.prototype
+         *
+         * @type {Event}
+         * @readonly
+         */
+        terrainProviderChanged : {
```

Comment:
While I'm all for reducing the API surface area, I'm not sure having the event only on the scene makes sense here.  We should stay consistent with what we've done elsewhere and we made a choice to have the globe be a separate documented class and since the property is there publicly, the event should be too.

Whenever we get around to doing a large API cleanup, then I could see us making a lot of these things private; but we shouldn't just start now because it will make the API even more disjoint.